app/scripts/modules/core/src/presentation/stepWizard/StepWizard.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+export interface IStepWizardInjectedProps {
+  currentIndex: number;
+  currentStep: React.ReactNode;
+  goToStep: (i: number) => void;
+}
+
+export interface IStepWizardProps {
+  steps: React.ReactNode[];
+  render: (props: IStepWizardInjectedProps) => React.ReactNode;
+}
+
+export const StepWizard = ({ render, steps }: IStepWizardProps) => {
```

Comment:
@christopherthielen I am finally revisiting this. Here is the very simple `StepWizard` we talked about a couple weeks ago. 

Given how simple it is, my preference would be to either make this an HOC (rather than using a render prop) to inject the `currentStep`, `currentIndex`, and `goToStep`, or to just abandon this and go back to my original plan of just creating the `ModalStepWizard`.

[This commit](https://github.com/spinnaker/deck/pull/8741/commits/11a5c7f69ce6bda369fcb8d5c9b4e216252702d5#diff-4c5869ab49ace445f582d1d129efba599b1a09942f5d2bc01260bfaf2414031cR26) has an example of the HOC, but it is less abstracted than this iteration.